File: mycollab-web/src/main/java/com/esofthead/mycollab/module/crm/events/CrmSettingEvent.java
package com.esofthead.mycollab.module.crm.events;

import com.esofthead.mycollab.eventmanager.ApplicationEvent;

public class CrmSettingEvent {

	public static class GotoNotificationSetting extends ApplicationEvent {
		private static final long serialVersionUID = 1L;

		public GotoNotificationSetting(Object source, Object data) {
			super(source, data);
		}
	}

	public static class GotoCustomViewSetting extends ApplicationEvent {
		private static final long serialVersionUID = 1L;

		public GotoCustomViewSetting(Object source, Object data) {
			super(source, data);
		}
	}
}

File: mycollab-web/src/main/scala/com/esofthead/mycollab/module/crm/events/CrmSettingEvent.scala
package com.esofthead.mycollab.module.crm.events

import com.esofthead.mycollab.eventmanager.ApplicationEvent

/**
  * @author MyCollab Ltd.
  * @since 5.0.3
  */
object CrmSettingEvent {

  class GotoNotificationSetting(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoCustomViewSetting(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}
